modules/src/ics03_connection/handler/conn_open_try.rs

```diff
@@ -116,14 +117,29 @@ mod tests {
             want_pass: bool,
         }
 
-        let msg_conn_try =
-            MsgConnectionOpenTry::try_from(get_dummy_msg_conn_open_try(10, 34)).unwrap();
-        let context = MockContext::new(10, Height::new(0, 35));
+        let host_chain_height = Height::new(0, 35);
+        let context = MockContext::new(host_chain_height);
+        let trusting_period = context.host_chain_history_size() as u64;
```

Comment:
Is there an effort to move away from the term "trusting period" towards "pruning"? I'm asking because I'm not sure if they are the same concept but just different names.

Reply:
`trusting period` is configured per light client and used when verifying light blocks, in the relayer's light client and the IBC ics02 client that runs on-chain. Its type is `Duration`. `pruning_window` is the number of states stored on a chain. The pruning strategy is configured in `app.toml`
`default: the last 100 states are kept in addition to every 500th state; pruning at 10 block intervals`

Reply:
Thanks for the explanation! This wasn't clear to me.